README.rst

```diff
@@ -71,6 +71,19 @@ You can also mount devices.
         mp.remount('rw,sync,guest') # remount with different options
         h.fs.touch('%s/new_file' % mp.target) # touch file
 
+Firewall 
+~~~~~~~~
+
+Allows to manage firewall configurarion. Check which firewall service is 
+running on host (firewalld/iptables) and make configure this service.
+
+  .. code:: python  
```

Comment:
I can seesome wird characters    at the start and end.. same at line 74, probably some file encoding problem?